drivers/gles2/shader_gles2.cpp

```diff
@@ -180,6 +181,14 @@ ShaderGLES2::Version *ShaderGLES2::get_current_version() {
 	strings.push_back("#define USE_HIGHP_PRECISION\n");
 #endif
 
+#ifdef ANDROID_ENABLED
```

Comment:
Should it really be Android-specific? If so, instead of exposing it as a `.mobile` override, it could be an `.Android` override.

Reply:
I don't have experience with iOS and don't know if the shader compiler would support it. I made it android specific now by moving it to .Android

Reply:
The `#ifdef` can be removed.